docker_explorer/lib/container.py

```diff
@@ -204,3 +215,41 @@ def GetHistory(self, show_empty_layers=False):
         else:
           history_str += 'Empty layer'
     return history_str
+
+  def _SetStorage(self, storage_name):
+    """Sets the storage_object attribute.
+
+    Args:
+      storage_name (str): the name of the storage.
+    Returns:
+      BaseStorage: a storage object.
+    Raises:
+      BadContainerException: if no storage Driver is defined, or if it is not
+        implemented
+    """
+    storage_class = self.STORAGES_MAP.get(storage_name, None)
+
+    if storage_class is None:
+      raise errors.BadContainerException(
+          'Storage driver {0} is not implemented'.format(storage_name))
+
+    self.storage_object = storage_class(
+        self.docker_directory, self.docker_version)
+
+  def Mount(self, mount_dir):
+    """Mounts the specified container's filesystem.
+
+    Args:
+      mount_dir (str): the path to the destination mount point
+    """
+
+    commands = self.storage_object.MakeMountCommands(self, mount_dir)
+    for c in commands:
+      print(c)
+    print('Do you want to mount this container Id: {0:s} on {1:s} ?\n'
+          '(ie: run these commands) [Y/n]'.format(self.container_id, mount_dir))
+    choice = raw_input().lower()
+    if not choice or choice == 'y' or choice == 'yes':
```

Comment:
`if not choice or or choice in ['y', 'yes']` or even `if choice in ['y', 'yes', '']`

Reply:
done